src/main/java/karuna/karuna_backend/security/config/WebSecurityConfig.java

```diff
@@ -48,6 +48,7 @@ private AuthorizeHttpRequestsConfigurer<HttpSecurity>.AuthorizationManagerReques
         return configurePublicHttpRequests(auth)
                 .requestMatchers(HttpMethod.GET, "/contents/**")
                 .permitAll()
+                .requestMatchers(HttpMethod.GET, "/visitors/messages").hasRole("ADMIN")
```

Comment:
No!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
In controller we have security right now!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!